config/authorities/format.yml

```diff
@@ -39,10 +39,14 @@ terms:
     term: DVCPRO
   - id: DVD
     term: DVD
-  - id: "Film: 16mm"
-    term: "Film: 16mm"
-  - id: "Film: 35mm"
-    term: "Film: 35mm"
+  - id: >
+      Film: 16mm
+    term: >
+      Film: 16mm
+  - id: >
+      Film: 35mm
+    term: >
```

Comment:
@sroosa any reason we are using `>` in here?

Reply:
I didn't want the the quotes, but if you don't use the > then it doesn't like that the : in the term. If you prefer the other way, I can deal with it.